tests/test_cli/test_core.py

```diff
@@ -23,14 +23,11 @@
 import pytest
 
 from aea.cli import cli
-from aea.cli.core import (
-    _try_add_key,
-    _try_generate_wealth,
-    _try_get_address,
-    _try_get_balance,
-    _try_get_wealth,
-    _wait_funds_release,
-)
+from aea.cli.add_key import _try_add_key
+from aea.cli.generate_wealth import _try_generate_wealth, _wait_funds_release
+from aea.cli.get_address import _try_get_address
```

Comment:
I think test files should mirror files in the modules being tested, so I suggest splitting up